Gemfile

```diff
@@ -9,10 +9,9 @@ gem 'rails', '~> 5.2.1'
 gem 'bcrypt', '3.1.12'
 
 # Use SCSS for stylesheets
-gem 'sass-rails', '~> 5.0'
-
-gem "bootstrap_form", "~> 2.7"
-gem "bootstrap-sass"
```

Comment:
`bootstrap-sass` only supports up to version3, after which they recommend using the `bootstrap` gem